interface/weno/drug-drug.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/** Copyright (C) 2016 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+ 
+$fake_register_globals=false;
+$sanitize_all_escapes=true;
+require_once("../globals.php");
+
+/*
+*   check to see if RxNorm installed
+*/ 
+	$rxn = sqlQuery("SELECT table_name FROM information_schema.tables WHERE table_name = '" . mitigateSqlTableUpperCase('RXNCONSO');
```

Comment:
remove lead spacing on this line.
Also, did you test this (hint, it will throw a php parsing error)